src/utils/validation/evaluation-administration-schema.ts

```diff
@@ -61,32 +55,26 @@ export const createEvaluationAdministrationSchema = object().shape({
 
       return new Date(start_date) <= new Date(end_date)
     })
-    .test(
-      "start-date",
-      "Must select a date that is within the valid range.",
```

Comment:
Don't remove this validation, this is for when range of date is more than or less than 50 years ago.

Reply:
Up